src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,75 @@
-export const Pagination = () => {};
+import React, { FC } from 'react';
```

Comment:
```suggestion
import { FC } from 'react';
```
remove import React or use React.FC amd remove import FC
it will fix such problem
![Screenshot from 2023-11-11 15-06-51](https://github.com/mate-academy/react_pagination/assets/33029864/40fb800a-d6cc-42d4-a266-ad74639b1588)